man/systemd.network.xml

```diff
@@ -2457,6 +2457,25 @@
         </listitem>
       </varlistentry>
 
+       <varlistentry>
+        <term><varname>CommonApplicationsKeptOverHead=</varname></term>
+        <listitem>
+          <para>Specifies the size of each packet on the wire may differ from that seen by Linux. Takes one of
+          ethernet, ether-vlan or ranges a number between -64 and 255. Setting ethernet equivalent to overhead 38 mpu 84 and
```

Comment:
Please use literal tag for ethernet and ether-vlan.

Reply:
"ranges a number" doesn#t look righ